### ISSUE TITLE
Import error when using xls2xform from command line (ref PR #153).

### ISSUE THREAD

**Issue by ivangayton** [XLSForm/pyxform#154]
Hi there,

There seems to be a problem using xls2xform.py from the command line in OSX and Ubuntu. I was able to fix it by removing all imports referring to the module pyxform, but when I submitted a pull request to that effect (#153) it failed the CI tests. I withdrew the pull request as a result of the failed tests.

Details:

I advised a friend to use PyXform to convert xlsforms to xml locally for testing, and when she cloned the repo and attempted to use the xls2xform.py script, the following error resulted:

```
  File "xls2xform.py", line 7, in <module>
    from pyxform import builder, xls2json
ImportError: No module named pyxform
```
This was on OSX. I pulled the latest code to my Ubuntu 16.04 machine, and found the same result: any attempt to run xls2xform, with or without arguments, generates this ImportError.

Looking through the code, it appears that a number of modules call for an import from the module pyxform. I'm not sure why it's been done this way; it appears to make it impossible to use the script directly from the command line in either OSX or Ubuntu, but perhaps there's another use-case for which this import structure works, or is necessary.

In any case, I've removed all import statements directly referencing the pyxform module, and it now works on both my Ubuntu machine and my friend's Mac.

After making these changes in my fork, calling:

```
python xls2xform path/to/infile.xlsx path/to/outfile.xml
```
Results in a success message and a working output file in the specified folder.

However, I have no idea if this somehow compromises other uses of the script! I haven't tested it on a Windows machine (I would think that Python is Python, but I don't know this, so safer not to asume). Furthermore, as this obviously breaks the CI tests, it's obviously problematic in some other way!

If it's necessary to keep the import calls to pyxform, I'd appreciate knowing why this is; perhaps I can try to learn what's necessary to make the changes that accommodate both that need and my desire to have a working command-line script to convert forms locally.

Thanks, as always, for creating and maintaining this extraordinary project, which I use on a daily basis for humanitarian and development work.

Ivan

**Comment by lognaturel** [comment]
@ivangayton I asked this on your PR but to make sure you read it -- did you try following the suggested steps in the README? You can either install from `pip` if you're not going to be modifying pyxform or [local install instructions](https://github.com/XLSForm/pyxform#running-pyxform-from-local-source). Note in particular that `xls2xform` is typically run from the root directory.

**Comment by ivangayton** [comment]
Hi @lognaturel , sorry I didn't respond to the comment on the PR; I did see it but have been in the field without time or Internet access!

Mea culpa: I did not read the instructions. I've been using pyxform for years, simply by calling the uninstalled script, and when I did a pull request and that stopped working, I failed to read the installation instructions, and simply assumed something was wrong.

Withdrawing the PR, and closing this issue, with apologies to all concerned!

Ivan
